modules/cli-extending-plugins-installing.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/extending-cli-plugins.adoc
+
+[id='cli-installing-plugins-{context}']
+= Installing and using CLI plug-ins
+
+Once you have written a custom plug-in for the {product-title} CLI, you can
+install it to use the additional functionality that it introduces.
+
+[IMPORTANT]
+====
+OpenShift CLI plug-ins are currently a Technology Preview feature.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+See the link:https://access.redhat.com/support/offerings/techpreview/[Red Hat
+Technology Preview features support scope] for more information.
+endif::[]
+====
+
+.Prerequisites
+
+* You must have the `oc` CLI tool installed.
+* You must have a CLI plug-in file that begins with `oc-` or `kubectl-`.
+
+.Procedure
+
+. If necessary, update the plug-in file to be executable.
++
+----
+$ chmod +x <plugin_file>
+----
+. Place the file anywhere in your `PATH`, such as `/usr/local/bin/`.
++
+----
+$ sudo mv <plugin_file> /usr/local/bin/.
+----
+. Run `oc plugin list` to make sure that the plug-in file is listed.
++
+----
+$ oc plugin list
+The following compatible plugins are available:
+
+/usr/local/bin/<plugin_file>
+----
++
+If your plug-in is not listed here, verify that the file name begins with `oc-`
+or `kubectl-`, is executable, and is on your `PATH`.
+. Invoke the new command or option introduced by the plug-in.
++
+For example, if you built and installed the `kubectl-ns` plug-in from the
+ link:https://github.com/kubernetes/sample-cli-plugin[Sample plug-in repository],
```

Comment:
We aren't supposed to include GitHub links. @vikram-redhat, will you PTAL?

Reply:
I think in this particular case it's worth mentioning both, since one is a sample cli plugin, and the other is a resource for plugin authors.

Reply:
Hmm, I do see that guidance here [1] now, but it doesn't explain why... this is the only site that is called out as unallowed. What's the reasoning @vikram-redhat? Looking through our source, I do see plenty of places where github is referenced, and it seems like it would make sense to do so (unless there's some issue I'm unaware of). In this particular case, I do think that this is valuable information for users if we can provide it.

[1] https://github.com/openshift/openshift-docs/blob/enterprise-4.0/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites

Reply:
@vikram-redhat - @kalexand-rh gave me some more background around where this "guidance" originated from. Let me know your thoughts on it, but maybe this is something we should discuss at the next team meeting.

Reply:
@bergerhoffer We have been phasing out unsupported GitHub links as we go along. For a new docset like 4.0, we need to get buy in from support delivery to make sure they are ok with us adding this link.

The rationale is that if a GitHub link is provided in product docs, it signals to the customers that the content/plugin from that link is Red Hat supported, which is not always the case.

I am happy to add this, if @sferich888 gives his approval, otherwise we need to take it out.